src/providers/personalData/store.tsx

```diff
@@ -107,13 +107,15 @@ export const usePersonalDataStore = createStore<PersonalDataStoreState, Personal
     persist: {
       key: 'personalData',
       whitelist: WHITELIST,
-      version: 0,
+      version: 0.1,
```

Comment:
So far we have been using only integer versions for store

```suggestion
      version: 1,
```